tests/adapters/switches/remote_test.py

```diff
@@ -476,18 +507,59 @@ def test_get_bonds(self):
         assert_that(if2.interface.access_vlan, equal_to(None))
         assert_that(if2.interface.trunk_native_vlan, equal_to(2999))
         assert_that(if2.interface.trunk_vlans, equal_to([3000, 3001, 3002]))
-        assert_that(if2.members, equal_to(['ge-0/0/1', 'ge-1/0/1']))
```

Comment:
It is  not  removed, there was a duplicate of this assert.
Le 2015-10-31 9:50 AM, "Martin Roy" <notifications@github.com> a écrit :

> In tests/adapters/switches/remote_test.py
> <https://github.com/internaphosting/netman/pull/32#discussion_r43571870>:
>
> > @@ -476,18 +507,59 @@ def test_get_bonds(self):
> >          assert_that(if2.interface.access_vlan, equal_to(None))
> >          assert_that(if2.interface.trunk_native_vlan, equal_to(2999))
> >          assert_that(if2.interface.trunk_vlans, equal_to([3000, 3001, 3002]))
> > -        assert_that(if2.members, equal_to(['ge-0/0/1', 'ge-1/0/1']))
>
> Why remove this test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/internaphosting/netman/pull/32/files#r43571870>.
>